wiki/Tournaments/MWC/2014/en.md

```diff
@@ -151,352 +142,347 @@ Groups
-| ![][flag_HK] Hong Kong          | 1     - **4** | **China** ![][flag_CN]          | [#1](https://osu.ppy.sh/mp/8583237) |
-| ![][flag_KR] South Korea        | 0     - **4** | **Thailand** ![][flag_TH]       | Win by default                |
-| ![][flag_MY] **Malaysia**       | **4** - 0     | United States ![][flag_US]      | [#1](https://osu.ppy.sh/mp/8583358) |
-| ![][flag_RU] Russian Federation | 0     - **4** | **China** ![][flag_CN]          | [#1](https://osu.ppy.sh/mp/8584000) |
-| ![][flag_ID] Indonesia          | 1     - **4** | **Japan** ![][flag_JP]          | [#1](https://osu.ppy.sh/mp/8584060) |
-| ![][flag_AU] **Australia**      | **4** - 0     | Philippines ![][flag_PH]        | Win by default                |
-| ![][flag_PH] Philippines        | 0     - **4** | **United States** ![][flag_US]  | Win by default                |
-| ![][flag_NO] Norway             | 0     - **4** | **Thailand** ![][flag_TH]       | Win by default                |
-| ![][flag_HK] **Hong Kong**      | **4** - 1     | Russian Federation ![][flag_RU] | [#1](https://osu.ppy.sh/mp/8585131) |
-| ![][flag_AU] **Australia**      | **0** - **4** | **Malaysia** ![][flag_MY]       | [#1](https://osu.ppy.sh/mp/8585801) |
-| ![][flag_ID] **Indonesia**      | **4** - 0     | France ![][flag_FR]             | [#1](https://osu.ppy.sh/mp/8585811) |
-| ![][flag_PH] Philippines        | 0     - **4** | **Germany** ![][flag_DE]        | Win by default                |
-| ![][flag_PT] Portugal           | 0     - **4** | **Japan** ![][flag_JP]          | Win by default                |
-| ![][flag_PL] Poland             | 0     - **4** | **China** ![][flag_CN]          | [#1](https://osu.ppy.sh/mp/8585840) |
-| ![][flag_MY] **Malaysia**       | **4** - 0     | Germany ![][flag_DE]            | [#1](https://osu.ppy.sh/mp/8586550) |
-| ![][flag_KR] **South Korea**    | **4** - 0     | Norway ![][flag_NO]             | Win by default                |
-| ![][flag_TH] **Thailand**       | **4** - 0     | United Kingdom ![][flag_GB]     | [#1](https://osu.ppy.sh/mp/8586635) |
-| ![][flag_JP] **Japan**          | **4** - 0     | France ![][flag_FR]             | [#1](https://osu.ppy.sh/mp/8586688) |
-| ![][flag_PL] Poland             | 0     - **4** | **Hong Kong** ![][flag_HK]      | [#1](https://osu.ppy.sh/mp/8586727) |
-| ![][flag_ID] **Indonesia**      | **4** - 0     | Portugal ![][flag_PT]           | Win by default                |
-| ![][flag_KR] **South Korea**    | **4** - 0     | United Kingdom ![][flag_GB]     | [#1](https://osu.ppy.sh/mp/8587506) |
-| ![][flag_AU] **Australia**      | **4** - 1     | Germany ![][flag_DE]            | [#1](https://osu.ppy.sh/mp/8587531) |
+| Team A | | | Team B | History |
+|---:|:---:|:---:|:---|:---:|
+| **Poland** ![][flag_PL]         | **4** | 2 | ![][flag_RU] Russian Federation | [#1](https://osu.ppy.sh/community/matches/8573871) |
```

Comment:
just throwing this here, the match links are dead 👻

Reply:
Well, even the old links were dead, but i updated all the links to follow the new linking (this was because the option to save the mp results forever wasnt activated back then).

Reply:
I knew that was the case... because @kj415j45 told me about it a few months ago. I was just mentioning this because it doesn't seem to make much sense to keep dead links.